.github/workflows/tests.yml

```diff
@@ -18,22 +18,22 @@ jobs:
     name: Tests
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.7, 3.10, 3.11]
```

Comment:
Please remove 3.7.

Reply:
Made the changes now it only refers to versions `3.10` and `3.11`